src/main/java/net/minecraftforge/fmp/capabilities/ICapabilityWrapper.java

```diff
@@ -0,0 +1,19 @@
+package net.minecraftforge.fmp.capabilities;
+
+import java.util.Collection;
+
+import net.minecraftforge.common.capabilities.Capability;
+
+/**
+ * Interface that allows mods to provide an implementation of a capability in the case that there's more than one implementation
+ * for one side of a block (multiparts).
+ * 
+ * @see Capability
+ * @see CapabilityWrapperRegistry
+ */
+public interface ICapabilityWrapper<T>
+{
```

Comment:
Yes. This should be a part of the capability system.

ICapabilityWrapper lets you combine multiple capabilities into one which is used, for example, by multiparts to return all capabilities accessible from a given side - but other uses are possible, such as a computer mod exposing the combined capabilities of peripheral ItemStacks.